### PR TITLE
Feature/hooray

### PR DESCRIPTION
**Pull request by Markuson**

### CODE REVIEW

File: staff/manuel-barzi/array-functions/2/every.js [hunk 1]
```diff
@@ -10,7 +10,11 @@ function every(array, callback) {
     if (!(array instanceof Array)) throw TypeError(array + ' is not an array');
     if (typeof callback !== 'function') throw new TypeError(callback + ' is not a function');
     
+<<<<<<< Updated upstream:staff/manuel-barzi/array-functions/2/every.js
```
Comment (edfrias) [inline staff/manuel-barzi/array-functions/2/every.js]: @Markuson careful! You have a merge conflict here! solve it before merge

File: staff/manuel-barzi/array-functions/2/for-each.js [hunk 2]
```diff
@@ -7,7 +7,11 @@
 function forEach(array, callback) {
 	if (!(array instanceof Array)) throw TypeError(array + ' is not an array');
 	if (typeof callback !== 'function') throw new TypeError(callback + ' is not a function');
+<<<<<<< Updated upstream:staff/manuel-barzi/array-functions/2/for-each.js
```
Comment (edfrias) [inline staff/manuel-barzi/array-functions/2/for-each.js]: @Markuson careful! You have a merge conflict here! solve it before merge

File: staff/marc-uson/youtube-channel-template/index.html [hunk 3]
```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
```
Comment (edfrias) [inline staff/marc-uson/youtube-channel-template/index.html]: @Markuson I'm not sure if these files should be here, it appears to me that you had a wrong folder while switching between branches and this "te la ha liao parda" :/

**Comment by lexbarba** [comment]
@Markuson some files should not be here. Please close this PR and create a new and clean one. Thanks.

**Pull request closed without merging**